lifting_veil/agents/rainbow_agent_new.py

```diff
@@ -61,8 +66,10 @@ def q_target_online(state):
   
   # Get the unweighted loss without taking its mean for updating priorities.
   (mean_loss, loss), grad = grad_fn(online_params, rng3, target, loss_weights)
+  updates, optimizer_state = optimizer.update(grad, optimizer_state)
+  online_params = optax.apply_updates(online_params, updates)
   optimizer = optimizer.apply_gradient(grad)
```

Comment:
remove

Reply:
I think I should remove just line 71.
You are using line 60 and 70: https://github.com/google/dopamine/blob/dabc84b1013b7ced981f189b2bb161e7fa94d31a/dopamine/jax/agents/rainbow/rainbow_agent.py#L87

Reply:
yes, the comment was for line 71. you need to keep 69 and 70.